Emby.AutoOrganize/Core/NameUtils.cs

```diff
@@ -78,25 +77,25 @@ private static string GetComparableName(string name)
         {
             name = name.RemoveDiacritics();
 
-            name = " " + name + " ";
+            name = ' ' + name + ' ';
```

Comment:
This isn't actually better since the chars will get boxed for the concat
(haven't benched it yet tho)